### PR TITLE
Added and fixed several Avocent device types

### PR DESCRIPTION
**Pull request by desnoe**

### CODE REVIEW

**Comment by danner26** [comment]
Please fix all tests @desnoe and then ping me once they are all passing. In the meantime I will revert this PR to a draft state

File: device-types/Avocent/ACS8008.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+manufacturer: Avocent
+model: ACS 8008
+slug: acs8008
```
Comment (danner26) [inline device-types/Avocent/ACS8008.yaml:4]: Please append `avocent-` to the slug

File: device-types/Avocent/ACS8016.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+manufacturer: Avocent
+model: ACS 8016
+slug: acs8016
```
Comment (danner26) [inline device-types/Avocent/ACS8016.yaml:4]: Please append `avocent-` to the slug

File: device-types/Avocent/ACS8032.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+manufacturer: Avocent
+model: ACS 8032
+slug: acs8032
```
Comment (danner26) [inline device-types/Avocent/ACS8032.yaml:4]: Please append `avocent-` to the slug

File: device-types/Avocent/ACS8048.yaml [hunk 4]
```diff
@@ -1,7 +1,8 @@
 ---
 manufacturer: Avocent
-model: ACS8048
+model: ACS 8048
 slug: acs8048
```
Comment (danner26) [inline device-types/Avocent/ACS8048.yaml:4]: Please append `avocent-` to the slug

File: device-types/Avocent/ACS8008.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+part_number: ACS-8008
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Avocent/ACS8008.yaml:7]: Please add a comment in markdown format with a link to the datasheet if you can. I tried to find one but I am seeing all different model numbers that don't exactly match (i.e. they have more to their names).

Comment (danner26) [inline device-types/Avocent/ACS8008.yaml:7]: Please add `weight` if you have it

Comment (danner26) [inline device-types/Avocent/ACS8008.yaml:7]: Please add `airflow` if you have it

File: device-types/Avocent/ACS8016.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+part_number: ACS-8016
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Avocent/ACS8016.yaml:7]: Please add `airflow` if you have it

File: device-types/Avocent/ACS8032.yaml [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+part_number: ACS-8032
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Avocent/ACS8032.yaml:7]: Please add `airflow` if you have it

File: device-types/Avocent/ACS8048.yaml [hunk 8]
```diff
@@ -1,7 +1,8 @@
...
+part_number: ACS-8048
```
Comment (danner26) [inline device-types/Avocent/ACS8048.yaml:5]: Please add `airflow` if you have it

File: device-types/Avocent/ACS8048.yaml [hunk 9]
```diff
@@ -1,7 +1,8 @@
...
 u_height: 1
 is_full_depth: false
```
Comment (danner26) [inline device-types/Avocent/ACS8048.yaml:7]: Please add `airflow` if you have it

Comment (danner26) [inline device-types/Avocent/ACS8048.yaml:7]: Please add `weight` if you have it

File: device-types/Avocent/ACS8032.yaml (hunk 7 above)
Comment (danner26) [inline device-types/Avocent/ACS8032.yaml:7]: Please add `weight` if you have it

File: device-types/Avocent/ACS8016.yaml (hunk 6 above)
Comment (danner26) [inline device-types/Avocent/ACS8016.yaml:7]: Please add `weight` if you have it

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Avocent/ACS8016.yaml (hunk 6 above)
Comment (danner26) [inline device-types/Avocent/ACS8016.yaml:7]: Please add a comment in markdown format if you have a link to the manufacturer provided datasheet

File: device-types/Avocent/ACS8032.yaml (hunk 7 above)
Comment (danner26) [inline device-types/Avocent/ACS8032.yaml:7]: Please add a comment in markdown format if you have a link to the manufacturer provided datasheet

File: device-types/Avocent/ACS8048.yaml (hunk 8 above)
Comment (danner26) [inline device-types/Avocent/ACS8048.yaml:5]: Please add a comment in markdown format if you have a link to the manufacturer provided datasheet

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Avocent/ACS8008.yaml (hunk 1 above)
Comment (desnoe) [reply device-types/Avocent/ACS8008.yaml]: done

File: device-types/Avocent/ACS8032.yaml (hunk 3 above)
Comment (desnoe) [reply device-types/Avocent/ACS8032.yaml]: done

File: device-types/Avocent/ACS8048.yaml (hunk 4 above)
Comment (desnoe) [reply device-types/Avocent/ACS8048.yaml]: done

File: device-types/Avocent/ACS8016.yaml (hunk 2 above)
Comment (desnoe) [reply device-types/Avocent/ACS8016.yaml]: done

File: device-types/Avocent/ACS8032.yaml (hunk 7 above)
Comment (desnoe) [reply device-types/Avocent/ACS8032.yaml:7]: done

File: device-types/Avocent/ACS8048.yaml (hunk 9 above)
Comment (desnoe) [reply device-types/Avocent/ACS8048.yaml:7]: done

File: device-types/Avocent/ACS8032.yaml (hunk 7 above)
Comment (desnoe) [reply device-types/Avocent/ACS8032.yaml:7]: done

File: device-types/Avocent/ACS8016.yaml (hunk 6 above)
Comment (desnoe) [reply device-types/Avocent/ACS8016.yaml:7]: done

File: device-types/Avocent/ACS8008.yaml (hunk 5 above)
Comment (desnoe) [reply device-types/Avocent/ACS8008.yaml]: There are several models of type 8008, so I refined the definition of one of the model + added link to the description

Comment (desnoe) [reply device-types/Avocent/ACS8008.yaml]: added

File: device-types/Avocent/ACS8016.yaml (hunk 6 above)
Comment (desnoe) [reply device-types/Avocent/ACS8016.yaml]: added

File: device-types/Avocent/ACS8032.yaml (hunk 7 above)
Comment (desnoe) [reply device-types/Avocent/ACS8032.yaml]: There are several models of type 8032, so I refined the definition of one of the model + added link to the description

File: device-types/Avocent/ACS8048.yaml (hunk 8 above)
Comment (desnoe) [reply device-types/Avocent/ACS8048.yaml]: There are several models of type 8048, so I refined the definition of one of the model + added link to the description

**Comment by desnoe** [comment]
@danner26 everything should be ok now

**Comment by danner26** [comment]
Hello, unfortunately today we had to make a change to the schema in which weight is defined by. The upstream NetBox API requires weight to be passed to it in the following format:
```
weight: 12.01
weight_unit: lb
```

Due to this change, I have updated your branch against the current master. This might require you to update your PR, if you had weight currently defined, so that your PR is in-line with the new weight schema. If you have any issues, please let me know or refer to [this PR](https://github.com/netbox-community/devicetype-library/pull/1187) for more information.

**Review by danner26 — APPROVED** [review]

**Pull request merged**